ui_framework/.babelrc

```diff
@@ -1,3 +0,0 @@
-{
-  "presets": ["react", "@spalger/babel-presets"]
-}
```

Comment:
Yep, even after completely reinstalling my node_modules, this problem persists unless I add the babelrc file back.